src/bin/pumpkindb-doctests.rs

```diff
@@ -23,10 +23,10 @@ use std::sync::mpsc;
 use regex::Regex;
 use glob::glob;
 
-
 use pumpkindb::script::{RequestMessage, ResponseMessage, EnvId, Env, Scheduler};
 use pumpkindb::script::{textparser, binparser};
 use pumpkindb::pubsub;
+use pumpkindb::database;
```

Comment:
Nitpick: I tried to avoid using "database" term to avoid having a database in the database (of course, we can't avoid the "lmdb" database term much, although it can be aliased on import), that's why I went for mod_storage vs mod_database.